docs/events.md

```diff
@@ -20,6 +20,8 @@ Events that TrenchBoot appeared on.
 * Linux Plumbers Conference 2022
     * Video: [TrenchBoot Update](https://www.youtube.com/watch?v=FFh3fWvVv0o)
     * [Slides](slides/TrenchBoot%20-%20LPC%202022%20-%20Final.pdf)
+* FOSDEM 2021
+    * Video: [Secure Upgrades with DRTM](https://video.fosdem.org/2021/D.firmware/firmware_suwd.webm)
```

Comment:
I could not find the slides for this presentation

Reply:
I think it would be best to do as @pietrushnic said and move it to a seperate PR

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/34